Whois/whois_module/whois_action.py

```diff
@@ -32,27 +33,30 @@ def time_list_tool(obj):
 
 class WhoisAction(Action):
     def run(self, arguments):
-        whois_result = whois.whois(arguments["query"])
-        return {
-            "Domain": {
-                "Name": str(list_tool(whois_result.domain_name, list, 0)),
-                "Whois": {
-                    "Domain": str(list_tool(whois_result.domain_name, list, 0)),
-                    "DomainStatus": whois_result.status,
-                    "DNSSec": str(whois_result.dnssec),
-                    "Raw": str(whois_result.text),
-                    "NameServers": whois_result.name_servers,
-                    "CreationDate": str(time_list_tool(whois_result.creation_date)),
-                    "UpdatedDate": str(time_list_tool(whois_result.updated_date)),
-                    "ExpirationDate": str(time_list_tool(whois_result.expiration_date)),
-                    "Registrar": {
-                        "Name": str(whois_result.registrar),
-                        "AbuseEmail": str(list_tool(whois_result.emails, list, 0)),
+        try:
+            whois_result = whois.whois(arguments["query"])
```

Comment:
Can you define a method to clear the argument? In order to transform it in something usable for Whois.

E.g:
- https://raw.githubusercontent.com/path -> githubusercontent.com
- raw.githubusercontent.com/path -> githubusercontent.com
- raw.githubusercontent.com -> githubusercontent.com
- https://githubusercontent.com/path -> githubusercontent.com
- githubusercontent.com -> githubusercontent.com